### PR TITLE
Feature/df 479 job publication contact

### PR DESCRIPTION
**Pull request by 99999989**

### CODE REVIEW

File: alv-portal-ui/src/app/job-publication/job-publication-form/public-contact/public-contact.component.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, ValidationErrors, Validators } from '@angular/forms';
+import { Salutation } from '../../../shared/backend-services/shared.types';
+import { of } from 'rxjs';
+import { phoneInputValidator } from '../../../shared/forms/input/input-field/phone-input.validator';
+import { EMAIL_REGEX } from '../../../shared/forms/regex-patterns';
+import { I18nService } from '../../../core/i18n.service';
```
Comment (birom) [inline alv-portal-ui/src/app/job-publication/job-publication-form/public-contact/public-contact.component.ts]: Unused import

File: alv-portal-ui/src/app/job-publication/job-publication-form/contact/contact.component.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    this.parentForm.addControl('contact', this.contact);
+
+    this.setDefaultLanguageOption();
+  }
+
+  private setDefaultLanguageOption() {
+    this.i18nService.currentLanguage$.pipe(
+      take(1),
+      tap((lang: string) => {
+          this.contact.get('languageIsoCode').setValue(lang);
```
Comment (birom) [inline alv-portal-ui/src/app/job-publication/job-publication-form/contact/contact.component.ts]: I would prefer to set the initial value in the subscribe method.

File: alv-portal-ui/src/app/job-publication/job-publication-form/public-contact/public-contact.component.ts (hunk 1 above)
Comment (99999989) [reply alv-portal-ui/src/app/job-publication/job-publication-form/public-contact/public-contact.component.ts]: fixed

File: alv-portal-ui/src/app/job-publication/job-publication-form/contact/contact.component.ts (hunk 2 above)
Comment (99999989) [reply alv-portal-ui/src/app/job-publication/job-publication-form/contact/contact.component.ts]: fixed

**Pull request merged**